public/views/partials/components/organisms/reset-password.liquid

```diff
@@ -0,0 +1,59 @@
+---
+metadata:
+  name: Reset password form
+  params:
+    context: {}
+    action: '/authentication_links/create'
+    redirect_to: '/'
+    errors: {}
+    values: {}
+---
+{% liquid
+  assign context = context | default: params.context
+  assign action = action | default: params.action
+  assign redirect_to = redirect_to | default: params.redirect_to
+  assign errors = errors | default: params.errors
+  assign values = values | default: params.values
+%}
+<form action="{{action}}" method="post" data-tc="create-authentication-link">
+  <p>{{ 'app.account.reset_password.description' | t }}</p>
+
+    {% liquid
+    assign input_params = '{}' | parse_json | hash_merge: name: 'authentication_link[email]', required: true, type: 'email', value: params.values.email, classes: "w-full"
+    assign label_params = '{}' | parse_json | hash_merge: content: 'E-mail'
+    assign description_params = '{}' | parse_json | hash_merge: content: 'E-mail address of the user'
+    assign validation = '{}' | parse_json
+    if params.errors['email']
+      hash_assign validation['passed'] = false
+      hash_assign validation['messages'] = params.errors['email']
+    endif
+    assign fieldset_params = '{}' | parse_json | hash_merge: label_params: label_params, input_params: input_params, description_params: description_params, validation: validation, classes: "mb-6"
+    theme_render_rc 'components/molecules/formfield', params: fieldset_params
+  %}
+
+  {% liquid
+    assign input_params = '{}' | parse_json | hash_merge: name: 'authenticity_token', type: 'hidden', value: context.authenticity_token
+    theme_render_rc 'components/atoms/input', params: input_params
+  %}
+
+  {% liquid
+    assign input_params = '{}' | parse_json | hash_merge: name: 'redirect_to', type: 'hidden', value: redirect_to
+    theme_render_rc 'components/atoms/input', params: input_params
+  %}
+
+  {% if context.constants.VERIFY_HCAPTCHA == "true" %}
+    <fieldset>
+      <script src="https://hcaptcha.com/1/api.js" async defer></script>
+      <div class='h-captcha' data-sitekey="{{ context.hcaptcha_site_key }}"></div>
+
+      {% render 'theme/simple/field_error', errors: object.errors.hcaptcha %}
+    </fieldset>
+  {% elsif context.environment == 'staging' %}
+      <!-- hCaptcha has been disabled on staging environments -->
+  {% endif %}
```

Comment:
It seems that this is template related code. I think it would be better to set hcaptha by a parameter and render the error using the component library.

Reply:
yep, nice catch. it was on my list but forgot to rewrite it. Will do this next!

Reply:
resolved